security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/AuthenticationTestSuite.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.ts.security.oidcclient.mtls;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+public interface AuthenticationTestSuite {
```

Comment:
Could be renamed this interface to something more "meaningful", something like `KeycloakMtlsAuthN`, because all of these names are going to be used by "Polarion" in order to generate a report. 

So if for example `verifyAuthenticated` fails, we are going to have an alert like: 
`KeycloakMtlsAuthN.verifyAuthenticated` fails.

Reply:
Refactored, thanks!